src/macros.rs

```diff
@@ -33,7 +33,35 @@ macro_rules! handle_command {
             if command.is_ansi_code_supported() {
                 write_ansi_code!($writer, command.ansi_code())
             } else {
-                command.execute_winapi().map_err($crate::ErrorKind::from)
+                command
+                    .execute_winapi(&mut $writer)
+                    .map_err($crate::ErrorKind::from)
+            }
+        }
+        #[cfg(unix)]
+        {
+            write_ansi_code!($writer, $command.ansi_code())
+        }
+    }};
+}
+
+#[doc(hidden)]
+#[macro_export]
+macro_rules! handle_fmt_command {
```

Comment:
I would like to see a short description why this macro exists. Something in the lines of: "Offers the same functionality as queue but is used for fmt::Write"

Reply:
Comment added.